src/main/java/com/example/sinitto/auth/dto/KakaoTokenResponse.java

```diff
@@ -3,12 +3,15 @@
 
 import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import jakarta.validation.constraints.NotNull;
 
 @JsonNaming(value = PropertyNamingStrategies.SnakeCaseStrategy.class)
 public record KakaoTokenResponse(
         String accessToken,
         String refreshToken,
+        @NotNull
         int expiresIn,
+        @NotNull
         int refreshTokenExpiresIn
```

Comment:
음수의 경우의 수는 없으므로 @Positive 를 사용해보세요.
```suggestion
        @NotNull @Positive
        int expiresIn,
        @NotNull @Positive
        int refreshTokenExpiresIn
```

Reply:
넵 적용해보겠습니다!